src/components/Footer.jsx

```diff
@@ -41,34 +43,76 @@ class SiteFooter extends React.Component {
       intl,
     } = this.props;
     const showLanguageSelector = supportedLanguages.length > 0 && onLanguageSelected;
-    const { config } = this.context;
+    const config = getConfig();
 
     return (
-      <footer
-        role="contentinfo"
-        className="footer d-flex border-top py-3 px-4"
-      >
-        <div className="container-fluid d-flex">
-          <a
-            className="d-block"
-            href={config.LMS_BASE_URL}
-            aria-label={intl.formatMessage(messages['footer.logo.ariaLabel'])}
-          >
-            <img
-              style={{ maxHeight: 45 }}
-              src={logo || config.LOGO_TRADEMARK_URL}
-              alt={intl.formatMessage(messages['footer.logo.altText'])}
-            />
-          </a>
-          <div className="flex-grow-1" />
+      <div className="wrapper wrapper-footer">
+        <footer id="footer" className="tutor-container">
+          <div className="footer-top">
+            <div className="powered-area">
+              <ul className="logo-list">
+                <li>{intl.formatMessage(messages['footer.poweredby.text'])}</li>
+                <li>
+                  <a href="https://docs.tutor.overhang.io" rel="noreferrer" target="_blank">
+                    <Image
+                      src={`${config.LMS_BASE_URL}/static/indigo/images/tutor-logo.png`}
+                      alt={intl.formatMessage(messages['footer.tutorlogo.altText'])}
+                      width="57"
+                    />
+                  </a>
+                </li>
+                <li>
+                  <a href="https://open.edx.org" rel="noreferrer" target="_blank">
+                    <Image
+                      src={logo || `${config.LMS_BASE_URL}/static/indigo/images/openedx-logo.png`}
+                      alt={intl.formatMessage(messages['footer.logo.altText'])}
+                      width="79"
+                    />
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <nav className="nav-colophon" aria-label="About">
+              <ol>
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/about`}>About Us</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/blog`}>Blog</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/donate`}>Donate</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/tos`}>Terms of Sevice</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/privacy`}>Privacy Policy</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/help`}>Help</a>
+                </li>
+
+                <li>
+                  <a href={`${config.LMS_BASE_URL}/contact`}>Contact Us</a>
```

Comment:
Not every platform will have all those pages. Thus, we should make the list of pages configurable in some way. I realise it may be too late for that, but can we add this as a future action item?

Reply:
Yes, We have to find a way to link the footer links from `tutor-indigo` configuration to here. For now, Should I remove these links and add them to future action item?